spec/controllers/queue_items_controller_spec.rb

```diff
@@ -123,14 +123,30 @@
   describe "POST update_queue" do
     it_behaves_like "requires sign in" do
       let(:action) do
-        post :update_queue, queue_items: [{id: 2,  position: 3}, {id: 1, position: 1}]
+        post :update_queue, queue_items: [
+         { id: 2,  position: 3 }, { id: 1, position: 1 }
```

Comment:
Unnecessary spacing detected.<br>Use 2 spaces for indentation in an array, relative to the start of the line where the left bracket is.